src/tests/nativeaot/SmokeTests/UnitTests/Delegates.cs

```diff
@@ -50,8 +50,10 @@ public static int Run()
             Console.WriteLine("Failed");
             result = Fail;
         }
-
+// Dynamic code generation not supported
```

Comment:
This should use the expression interpreter like on all other native AOT targets. Should this be tracked as a bug instead?

Reply:
I believe the problem is related to the way `System.Linq.Expressions.dll` is built for iOS-like platforms which I am trying to fix via: 
https://github.com/dotnet/runtime/pull/86758 
https://github.com/dotnet/runtime/pull/86971

In general, this was called out by @MichalStrehovsky here: https://github.com/dotnet/runtime/issues/69410#issuecomment-1149334036 
based on his work: https://github.com/dotnet/runtime/pull/61952

I can open a tracking issue to reference this.

Reply:
This is being tracked in: https://github.com/dotnet/runtime/issues/87924